src/Frontend/components/app/layout/AppMenu.js

```diff
@@ -10,21 +10,18 @@
 
 'use strict';
 
-import React from 'react';
-import MenuWrapper from './menu/MenuWrapper';
-import Menu from './menu/Menu';
+import React             from 'react';
+import MenuWrapper       from './menu/MenuWrapper';
+import Menu              from './menu/Menu';
+import menu from '../../../config/menu';
+import {connect}      from 'react-redux';
+import *  as menuActions from '../../../actions/menuActions';
+import {bindActionCreators} from 'redux';
 
-/**
- * Component which simplifies the usage of the menu API.
- *
- * @param {Object} props The config for this wrapper component.
- *
- * @returns {React.Element} The markup of the menu.
- */
-const AppMenu = props => {
+const AppMenu = ({items, actions}) => {
   return (
-    <MenuWrapper items={props.config}>
-      <Menu />
+    <MenuWrapper>
+      <Menu rawItems={menu} items={items} actions={actions}/>
```

Comment:
I know we discussed about it already. But I think it's better to load the `menu` config in the `Menu` component directly  and remove the `rawItems` property again (passing almost the same stuff two times isn't a good solution IMHO)